app/assets/javascripts/govuk_publishing_components/components/feedback.js

```diff
@@ -68,45 +70,71 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
         setHiddenValuesNotUsefulForm(gaClientId)
       })
 
-      $element.find('form').on('submit', function (e) {
-        e.preventDefault()
-        var $form = $(this)
-        $.ajax({
-          type: 'POST',
-          url: $form.attr('action'),
-          dataType: 'json',
-          data: $form.serialize(),
-          beforeSend: disableSubmitFormButton($form),
-          timeout: 6000
-        }).done(function (xhr) {
-          trackEvent(getTrackEventParams($form))
-          showFormSuccess(xhr.message)
-          revealInitialPrompt()
-          setInitialAriaAttributes()
-          that.$activeForm.toggleClass(jshiddenClass)
-        }).fail(function (xhr) {
-          showError(xhr)
-          enableSubmitFormButton($form)
-        })
-      })
+      // much of the JS needed to support sending the form contents via this script is
+      // unsupported by IE, even IE11. This check causes IE to not intercept form submits
+      // and let them happen normally, which is handled already by the backend
```

Comment:
I think that this is a good approach to let IE11 fall back to the non-JavaScript version of the component.

It's good to document this in the code - and it'd be good to think about how to let the component guide show the browser support for each component, much like the accessibility criteria.

Reply:
That is a good idea. I think doing that properly is outside the scope of this PR, but I'll add a note to the YML file to make this clear.

Reply:
Super - ta!